examples/twittermap/noah/src/main/scala/edu/uci/ics/cloudberry/noah/TwitterJSONTagToADM.scala

```diff
@@ -36,14 +37,24 @@ object TwitterJSONTagToADM {
       case "-city" :: value :: tail => shapeMap += CityLevel -> value; parseOption(tail)
       case "-thread" :: value :: tail => threadNumber = value.toInt; parseOption(tail)
       case "-debug" :: value :: tail => isDebug = true; parseOption(tail)
+      case "-fileFormat" :: value :: tail => file = value; parseOption(tail)
       case option :: tail => System.err.println("unknown option:" + option); System.err.println(usage); System.exit(1);
     }
   }
 
-  def tagOneTweet(ln: String, usGeoGnosis: USGeoGnosis) = {
+  def tagOneTweet(ln: String, usGeoGnosis: USGeoGnosis) = synchronized {
```

Comment:
I don't think we need a `synchronized` for this function.

Reply:
Since we use multithread to ingest data, I think it is better to keep threads safe.

Reply:
But this way, the whole process is totally sequential now. Multithreading will not increase the throughput at all.

Reply:
Resolved in commit d678d5a468fc4a4a85da6f7a34eda0f4ca33a8f7